test/common/subsciption_install_plan.go

```diff
@@ -0,0 +1,76 @@
+package common
```

Comment:
Small typo in the file name :)

Reply:
Good spot ! I pushed the change now if you want to have another look :smile: